web/packages/teleterm/src/ui/Search/pickers/ResultList.tsx

```diff
@@ -119,7 +119,7 @@ export function ResultList<T>(props: ResultListProps<T>) {
             <InteractiveItem
               ref={isActive ? activeItemRef : null}
               role="menuitem"
-              $active={isActive}
+              active={isActive}
```

Comment:
>What's the convention behind adding a dollar sign to props?

https://styled-components.com/docs/api#transient-props

(it does not have any effect here, so I removed it)